sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -48,6 +48,19 @@ export interface AuthorizeRequestOptions {
     scopes: string[];
 }
 
+// @public
+export function auxiliaryAuthenticationPolicy(options: AuxiliaryAuthenticationPolicyNameOptions): PipelinePolicy;
```

Comment:
I updated the policy name as `auxiliaryAuthenticationHeaderPolicy`. Not only because other languages also take similar name but also I think this policy is more focusing on adding header `x-ms-authorization-auxiliary` not a general multi-tenant auth policy.

```typescript
export const auxiliaryAuthenticationHeaderPolicyName = "auxiliaryAuthenticationHeaderPolicy";
```